### PR TITLE
doctl-kubernetes-cluster: add page

### PR DESCRIPTION
**Pull request by tonytheleg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
doctl version 1.82.0-release

Addresses [Issue 7475](https://github.com/tldr-pages/tldr/issues/7475)
`doctl kubernetes` command has 3 main subcommands (`create`, `options`, and `1-click`). This covers the `cluster` subcommand

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/doctl-kubernetes-cluster.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# doctl kubernetes cluster
+
+> For managing Kubernetes clusters and viewing configuration options relating to clusters.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/kubernetes/cluster/>.
+
+- Create a Kubernetes cluster:
+
+`doctl kubernetes cluster create --count {{3}} --region {{nyc1}} --size {{s-1vcpu-2gb}} --version {{latest}} {{cluster-name}}`
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:8]:
```suggestion
`doctl kubernetes cluster create --count {{3}} --region {{nyc1}} --size {{s-1vcpu-2gb}} --version {{latest}} {{cluster_name}}`
```

File: pages/common/doctl-kubernetes-cluster.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`doctl kubernetes cluster list`
+
+- Fetch and save the kubeconfig:
+
+`doctl kubernetes cluster kubeconfig save {{cluster-name}}`
+
+- Check for available upgrades:
+
+`doctl kubernetes cluster get-upgrades {{cluster-name}}`
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:20]:
```suggestion
`doctl kubernetes cluster get-upgrades {{cluster_name}}`
```

File: pages/common/doctl-kubernetes-cluster.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Upgrade a cluster:
+
+`doctl kubernetes cluster upgrade {{cluster-name}}`
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:24]:
```suggestion
`doctl kubernetes cluster upgrade {{cluster_name}}`
```

File: pages/common/doctl-kubernetes-cluster.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Delete a cluster:
+
+`doctl kubernetes cluster delete {{cluster-name}}`
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:28]:
```suggestion
`doctl kubernetes cluster delete {{cluster_name}}`
```

File: pages/common/doctl-kubernetes-cluster.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`doctl kubernetes cluster create --count {{3}} --region {{nyc1}} --size {{s-1vcpu-2gb}} --version {{latest}} {{cluster-name}}`
+
+- List all Kubernetes clusters:
+
+`doctl kubernetes cluster list`
+
+- Fetch and save the kubeconfig:
+
+`doctl kubernetes cluster kubeconfig save {{cluster-name}}`
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:16]:
```suggestion
`doctl kubernetes cluster kubeconfig save {{cluster_name}}`
```

File: pages/common/doctl-kubernetes-cluster.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Check for available upgrades:
+
+`doctl kubernetes cluster get-upgrades {{cluster-name}}`
+
+- Upgrade a cluster:
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:22]: What upgrades?

File: pages/common/doctl-kubernetes-cluster.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# doctl kubernetes cluster
+
+> For managing Kubernetes clusters and viewing configuration options relating to clusters.
```
Comment (navarroaxel) [inline pages/common/doctl-kubernetes-cluster.md:3]:
```suggestion
> Manage Kubernetes clusters and view configuration options relating to clusters.
```

**Comment by tonytheleg** [comment]
@navarroaxel requested updates made. Thanks!

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**